README.md

```diff
@@ -1,10 +1,13 @@
 # Minesweeper
 
-is a single player puzzle game. The object of the game is to clear a rectangular board containing hidden "mines" or bombs without detonating any of them, using the guidance on the number of adjacent mines in each field. The game dates from the 1960s
+A single-player puzzle game. You need to clear a rectangular board containing hidden "mines" or bombs without detonating any of them, using the guidance on the number of adjacent mines in each field. The game dates from the 1960s
```

Comment:
Documents that are written in the second person are easier to read and encourage iteration.
https://developers.google.com/style/person